scss/grid/_layout.scss

```diff
@@ -317,38 +317,50 @@
         width: auto;
         display: block;
 
-        > span {
+        > span,
+        .k-filtercell-wrapper {
             padding-right: calc( 2 * (#{$button-calc-size} + #{$cell-padding-y / 2}));
             height: calc( #{$button-calc-size} );
             display: block;
             position: relative;
 
+            > label {
+                vertical-align: middle;
+            }
+
+        }
+
+        > span,
+        .k-filtercell-operator {
             > .k-button {
                 position: absolute;
                 top: 0;
                 right: 0;
             }
-            > label {
-                vertical-align: middle;
-            }
-
         }
+
+
         > .k-operator-hidden {
             padding-right: calc( #{$button-calc-size} + #{$cell-padding-y / 2});
         }
 
-        .k-widget {
+        .k-widget,
+        .k-filtercell-wrapper > .k-textbox {
             display: flex;
             width: auto;
             flex: 1;
         }
+
+        .k-filtercell-wrapper > .k-textbox {
+            width: 100%;
+        }
```

Comment:
There's a `width: auto` with the same selector on [line 350](https://github.com/telerik/kendo-theme-default/pull/262/files/80695152f353eefae8ba21dbe67ababd71051ade#diff-cee715f51ad80e6885203922176ce834R350)?